assets/js/base/components/product-list/container.js

```diff
@@ -1,47 +1,45 @@
 /**
  * External dependencies
  */
-import { Component } from 'react';
+import { useState } from '@wordpress/element';
 import PropTypes from 'prop-types';
-import withQueryStringValues from '@woocommerce/base-hocs/with-query-string-values';
 
 /**
  * Internal dependencies
  */
 import ProductList from './index';
 
-class ProductListContainer extends Component {
-	onPageChange = ( newPage ) => {
-		this.props.updateQueryStringValues( {
-			product_page: newPage,
-		} );
+const ProductListContainer = ( {
+	attributes,
+	// eslint-disable-next-line camelcase
+	product_page = 1,
+	// eslint-disable-next-line camelcase
+	product_sort,
```

Comment:
`product_page` and `product_sort` props were coming from `withQueryStringValues`, so they are not present anymore. Should we remove them from here?

Reply:
+1 to this, we should remove these as props, and the state should be initialised with default values. Or if we want to preserve as props in the component interface, rename to camelCase.